utils/symbolicate-linux-fatal.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python
+
+# TODOs:
+# * verbose output
+# * search symbols by name for the not <unavailable> ones
+
+import sys
+import lldb
```

Comment:
This LLDB is probably the system one, not the Swift-enabled LLDB (so using it is as good as any other tool).  Did you consider using `addr2line`, which is the usual way of symbolicating?